### PR TITLE
Webscan Requests: Server Overload CLI

### PR DESCRIPTION
**Pull request by payoub125**
### Problem
The precompiled Server Overload headers where too large for the CLI to process as instruction flags.

### Solution
To address this, I implemented a dedicated command specifically designed to handle Server Overload headers. This command accepts header keys and the desired header value size. It still leverages the general request function, with an additional layer to generate and inject the large headers.

### Note
The other Header Injection tools will still use the webscan requests command

### CODE REVIEW

File: cmd/requests.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+			}
+
+			a.OutputSignal.Content = report
+
+		},
+	}
+	serverOverloadCmd.Flags().StringSlice("headerName", []string{}, "Specifies Header keys to use in request.")
+	serverOverloadCmd.Flags().Int("headerSize", 100, "Specifies the length of header values to include in requests.")
+
+	_ = requestsCmd.MarkFlagRequired("headerNames")
```
Comment (sean-hacker) [inline cmd/requests.go]: Typo? I think it should be `headerName`

Comment (sean-hacker) [inline cmd/requests.go]: Typo? I think it should be `headerName`

Comment (payoub125) [reply cmd/requests.go]: Yep - thanks for the catch

Comment (payoub125) [reply cmd/requests.go]: Yep - Thanks for the catch

**Review by sean-hacker — APPROVED** [review]

**Pull request merged**